ui/css/utilities/fonts.scss

```diff
@@ -67,28 +67,28 @@ $font-path: './fonts';
   font-family: 'Euclid Circular B';
   font-style: normal;
   font-weight: 400;
-  src: url('#{$font-path}/Euclid/EuclidCircularB-Regular-WebXL.ttf') format('truetype');
+  src: url('#{$font-path}/Euclid/EuclidCircularB-Regular-WebXL.woff2') format('woff2');
 }
 
 @font-face {
   font-family: 'Euclid Circular B';
   font-style: italic;
   font-weight: 400;
-  src: url('#{$font-path}/Euclid/EuclidCircularB-RegularItalic-WebXL.ttf') format('truetype');
+  src: url('#{$font-path}/Euclid/EuclidCircularB-RegularItalic-WebXL.woff2') format('woff2');
 }
 
 @font-face {
   font-family: 'Euclid Circular B';
   font-style: normal;
   font-weight: 500;
-  src: url('#{$font-path}/Euclid/EuclidCircularB-Medium.ttf') format('truetype');
```

Comment:
The newest font package has a `EuclidCircularB-Medium-WebS.woff2` and a `EuclidCircularB-Medium-WebXL.woff2`. It doesn't have a `EuclidCircularB-Medium.woff2`, so I just went with the WebXL version because all of our other fonts also use that version.